index.js

```diff
@@ -339,16 +352,140 @@ class Raspi extends events.EventEmitter {
     }
   }
 
+  i2cConfig(delay) {
+    if (this[i2c] === undefined) {
```

Comment:
Oh I see, I didn't realize that i2c-bus didn't handle that for us.

Is there a problem with loading the I2C module at app startup? Unless there's a technical reason that prevents us from doing so, I think we should just load it anyways, whether or not it's going to be used. The J5 API kind of implies that I2C is always loaded and available anyways (given it's lack of I2C ready events, init methods, setting mode, etc)